acd-config-editor/docker_deploy_service.md

```diff
@@ -0,0 +1,48 @@
+# Watson Health Cognitive Services (WHCS) Docker Instructions
```

Comment:
Remove this file - in the pages source that deb has will have the docker install instructions.  we can have a README.md in this new folder that points to that doc on the published pages.